opentreemap/treemap/instance.py

```diff
@@ -112,6 +113,12 @@ def get_instance_permission_spec():
                              'of a tree or map feature'),
             'default_role_names': [Role.ADMINISTRATOR, Role.EDITOR],
             'label': _('Can View External Link')
+        },
+        {
+            'codename': PERMISSION_ACCESS_WORKS_MANAGEMENT,
+            'description': _('Can view or modify Works Management entities'),
+            'default_role_names': [Role.ADMINISTRATOR],
+            'label': _('Can Access Works Management')
```

Comment:
I now see that I capitalized the first letter of all the words because it's friend, `view_external_link` capitalizes all the words: `'Can View External Link'`.

Leaving them both as is for now. If we want greater consistency, between instance permissions and model permissions, that seems like a separate issue.